### PR TITLE
DOCSP-30586 Data Verification Drawer

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## Description

Data Verification drawer conversion from markdown to snooty.

## Staging

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30586/jobs/data-verification/data-verification/

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30586/jobs/data-verification/use-data-verification/

## Jira

https://jira.mongodb.org/browse/DOCSP-30586

## Build

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=648768af67a273b8169e20f9

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
@sarah-olson-mongodb Conversion ready for review.

File: source/jobs/data-verification/data-verification.txt [hunk 1]
```diff
@@ -2,6 +2,35 @@
...
+do not match.
+
+Data verification does not consider the following when verifying 
+destination databases:
+
+- The destination collection contained unrelated data before the sync 
+  job started.
+- The data in the relational database tables has changed since the sync 
+  job was started.
+
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/data-verification.txt:32]: (info typing)
Use a Learn More heading and then include just a link to the Use Data Verification page, omitting the additional text.

File: source/jobs/data-verification/use-data-verification.txt [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+
+To view status and metadata information about the data verification process
+for a sync job, click the :guilabel:`SYNC JOB / DATA VERIFICATION` slider.
+
+.. image:: /img/jobs/data-verification/data-verification-click-slider.png
+   :alt: Click Data Verification Slider
+
+Data Verification Overview
+--------------------------
+
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:44]: (typo) 
s/b "metadata"

File: source/jobs/data-verification/use-data-verification.txt [hunk 3]
```diff
@@ -1,3 +1,115 @@
...
+Enable Data Verification
+------------------------
+
+You can enable data verification when :ref:`creating a sync job <rm-create-jobs>`
+on the :guilabel:`Data Migration` tab.
+
+On the :guilabel:`Migration Options` module, click the 
+:guilabel:`Verify migrated data` check box for your sync job.
+
+
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:27]: (image)
is it possible to decrease the size of this image a bit.

File: source/jobs/data-verification/use-data-verification.txt [hunk 4]
```diff
@@ -1,3 +1,115 @@
...
+   * - Status
+     - Description
+   * - ABORTED
+     - A running data validation was unable to complete due to an error.
+   * - CANCELLED
+     - The data validation process was cancelled by clicking ``TERMINATE``.
+   * - CANCELLING
+     - The data validation process is being canceled.
+   * - FAILED
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:84]: (active voice)
s/b "...are listed on the Issues pane."

File: source/jobs/data-verification/use-data-verification.txt [hunk 5]
```diff
@@ -1,3 +1,115 @@
...
+When a sync job is successful the status updates to :guilabel:`VERIFIED`
+and a double checkmark is displayed in the :guilabel:`History` pane.
+
+.. image:: /img/jobs/data-verification/data-verification-double-checkmark.png
+   :alt: History pane double checkmark
+
+Data Verification Issues
+------------------------
+
+If the data verification process detects any differences between the
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:103]: (active voice)
s/b ".. the errors are reported ..."

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks a ton @ianf-mongodb. A few super minor items. My main question is: Can we reduce the size and/or number of the screenshots here. Would you mind think this through before we merge this one?

File: source/jobs/data-verification/data-verification.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/jobs/data-verification/data-verification.txt:32]: Updated.

File: source/jobs/data-verification/use-data-verification.txt (hunk 2 above)
Comment (ianf-mongodb) [reply source/jobs/data-verification/use-data-verification.txt:40]: Updated.

File: source/jobs/data-verification/use-data-verification.txt (hunk 4 above)
Comment (ianf-mongodb) [reply source/jobs/data-verification/use-data-verification.txt:80]: Updated.

File: source/jobs/data-verification/use-data-verification.txt (hunk 3 above)
Comment (ianf-mongodb) [reply source/jobs/data-verification/use-data-verification.txt]: I looked, I think the only thing you can control is the figwidth. I also think we can remove the image entirely.

**Review by ianf-mongodb — COMMENTED** [review]
@sarah-olson-mongodb Removed the oversized image and made other corrections.

File: source/jobs/data-verification/data-verification.txt [hunk 6]
```diff
@@ -2,6 +2,38 @@
 Data Verification
 =================
 
+Data verification is a process which is built into the sync engine.
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/data-verification.txt:5]: ([that vs. which](https://www.mongodb.com/docs/meta/style-guide/writing/use-that-which-correctly/)) 
Nit. Should have marked this the first time. S/b: "... is a process that is ..."

File: source/jobs/data-verification/use-data-verification.txt [hunk 7]
```diff
@@ -1,3 +1,112 @@
...
-=====================
\ No newline at end of file
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:12]: (that vs. which) 
Same as prior comment. "... separate process that is ..."

File: source/jobs/data-verification/use-data-verification.txt [hunk 8]
```diff
@@ -1,3 +1,112 @@
...
+To view status and metadata information about the data verification process
+for a sync job, click the :guilabel:`SYNC JOB / DATA VERIFICATION` slider.
+
+.. image:: /img/jobs/data-verification/data-verification-click-slider.png
+   :alt: Click Data Verification Slider
+
+Data Verification Overview
+--------------------------
+
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:40]: (punctuation)
"On the Data Verification pane[,] you can ..."

File: source/jobs/data-verification/use-data-verification.txt [hunk 9]
```diff
@@ -1,3 +1,112 @@
...
+   * - Status
+     - Description
+   * - ABORTED
+     - A running data validation was unable to complete due to an error.
+   * - CANCELLED
+     - The data validation process was cancelled by clicking ``TERMINATE``.
+   * - CANCELLING
+     - The data validation process is being canceled.
+   * - FAILED
+     - The source and destination databases do not match. The corresponding errors are listed on the ``Issues`` pane.
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:81]: (markup)
Issues should be guilabel

File: source/jobs/data-verification/use-data-verification.txt [hunk 10]
```diff
@@ -1,3 +1,112 @@
...
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Status
+     - Description
+   * - ABORTED
+     - A running data validation was unable to complete due to an error.
+   * - CANCELLED
+     - The data validation process was cancelled by clicking ``TERMINATE``.
```
Comment (sarah-olson-mongodb) [inline source/jobs/data-verification/use-data-verification.txt:77]: (markup)
Terminate s/b guilabel.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @ianf-mongodb. Couple quick additional fixes. (Sorry about not catching them the first time.) After that, LGTM!

**Comment by ianf-mongodb** [comment]
Thanks @sarah-olson-mongodb Updated and merging this into pre-local.

**Pull request merged**